adam/situation/templates/phase1_templates.py

```diff
@@ -189,13 +197,31 @@ def generate_situations(
                 for obj_var in template.object_variables
             )
 
+            always_relations = []
```

Comment:
@LichMaster98 : We don't want to special case this logic in for the deadline hack; you can copy it somewhere for when we address this for real.  We are ok with just having the size relations missing for now.